### ISSUE TITLE
Does not support Tailwind v3

### ISSUE THREAD

**Issue by AliMosbah** [kenhyuwa/litepie-datepicker#79]
There is a problem with support for Tailwind v3 The package is not compatible

**Comment by marcotas** [comment]
Yeah the npm package is still incompatible with Tailwind 3. Any chance to update the NPM package? Or how can we use the version compatible with Tailwind v3?

<img width="738" alt="image" src="https://user-images.githubusercontent.com/3236791/193604878-0cab68ec-6c14-4fcd-9d16-cd88787412ca.png">

**Comment by nirgn975** [comment]
I recommend using [vue-tailwind-datepicker](https://github.com/elreco/vue-tailwind-datepicker). It's has the same features and it's updated to vue3 and tailwind3.

**Comment by Valar103769** [comment]
> I recommend using [vue-tailwind-datepicker](https://github.com/elreco/vue-tailwind-datepicker). It's has the same features and it's updated to vue3 and tailwind3.

too Stuck and stopped

**Comment by elreco** [comment]
> too Stuck and stopped

Hi 😄  if you got any issue please let me know

**Comment by Valar103769** [comment]
> > too Stuck and stopped
> 
> Hi 😄 if you got any issue please let me know

不知道你改了什么，反正很卡，点一下要等一秒才弹出overlay

**Comment by jervis446** [comment]
Not working for me also, any other alternative ??

**Comment by sentiasa** [comment]
Hi @kenhyuwa

I think you haven't tagged the release (1.0.15) that's why we can't download it.
Also, tried `@next` but seeing the same error.

I think if you tag the main branch as 1.0.15, we'll be able to use it.

```
$ npm install -S litepie-datepicker@next

npm ERR! code ETARGET
npm ERR! notarget No matching version found for litepie-datepicker@next.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bertug/.npm/_logs/2023-03-06T15_01_17_932Z-debug-0.log
```

-------
```
$ npm install -S litepie-datepicker@1.0.15

npm ERR! code ETARGET
npm ERR! notarget No matching version found for litepie-datepicker@1.0.15.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bertug/.npm/_logs/2023-03-06T15_03_09_662Z-debug-0.log
```

**Comment by sentiasa** [comment]
> I recommend using [vue-tailwind-datepicker](https://github.com/elreco/vue-tailwind-datepicker). It's has the same features and it's updated to vue3 and tailwind3.

I moved to vue-tailwind-datepicker too, it was a swift change and didn't require anything extra. Thanks @nirgn975
